plugins/modules/packaging/language/pear.py

```diff
@@ -150,9 +180,39 @@ def remove_packages(module, packages):
     module.exit_json(changed=False, msg="package(s) already absent")
 
 
-def install_packages(module, state, packages):
+def install_packages(module, state, packages, prompts):
     install_c = 0
-
+    has_prompt = True if prompts else False
+
+    if has_prompt:
+        nb_prompts = len(prompts)
+        nb_packages = len(packages)
+
+        if nb_prompts > 0 and (nb_prompts != nb_packages):
+            if nb_prompts > nb_packages:
+                diff = nb_prompts - nb_packages
+                msg = "%s packages to install but %s prompts to expect. %s prompts will be ignored" % (to_text(nb_packages), to_text(nb_prompts), to_text(diff))
+            else:
+                diff = nb_packages - nb_prompts
+                msg = "%s packages to install but only %s prompts to expect. %s packages won't be expected to have a prompt" \
+                    % (to_text(nb_packages), to_text(nb_prompts), to_text(diff))
+            module.warn(msg)
+
+        # Preparing prompts answer according to item type
+        tmp_prompts = []
+        default_prompt_answer = "\n"
+        for _item in prompts:
+            # If the current item is a dict then we expect it's key to be the prompt regex and it's value to be the answer
+            # We also expect here that the dict only has ONE key and the first key will be taken
+            if isinstance(_item, dict):
+                key = list(_item.keys())[0]
+                answer = _item[key] if _item[key] else default_prompt_answer
+                answer += "\n"
```

Comment:
If I set a prompt:
```
toto: ''
```
I will end up with answer="\n\n"

I suggest the following as `_item[key]` should  always be a string:
```suggestion
                answer = _item[key] + "\n"
```